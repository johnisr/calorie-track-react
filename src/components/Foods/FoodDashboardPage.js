import React from 'react';
import FoodListDisplayFoodsContainer from './FoodListDisplayFoods/FoodListDisplayFoodsContainer';
import FoodListFilters from '../FoodListFilters';
import FoodsDisplayTabs from './FoodsDisplayTabs/FoodsDisplayTabs';
import Header from '../Header/Header';
import { history } from '../../routers/AppRouter';

const FoodDashboardPage = () => (
  <div className="container">
  <div className="row">
    <section className="section__full-start-end">
      <Header history={history}/>
    </section>
    <section className="section__center-start-end">
      <FoodListFilters />
    </section>
    <section className="section__center-6-start-end">
      <FoodListDisplayFoodsContainer />
    </section>
    <section className="section__center-6-start-end margin-bottom-large">
      <FoodsDisplayTabs />
    </section>
  </div>
  </div>
);

export default FoodDashboardPage;