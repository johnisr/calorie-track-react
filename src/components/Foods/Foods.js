import React from 'react';
import FoodListContainer from './FoodListContainer/FoodListContainer';
import FoodListFilters from '../FoodListFilters';
import FoodsDisplayTabs from './FoodsDisplayTabs/FoodsDisplayTabs';
import Header from '../Header/Header';
import { history } from '../../routers/AppRouter';

const Foods = () => (
  <div className="container">
  <div className="row">
    <section className="section__full-start-end">
      <Header history={history}/>
    </section>
    <section className="section__center-start-end">
      <FoodListFilters />
    </section>
    <section className="section__center-6-start-end">
      <FoodListContainer />
    </section>
    <section className="section__center-6-start-end margin-bottom-large">
      <FoodsDisplayTabs />
    </section>
  </div>
  </div>
);

export default Foods;