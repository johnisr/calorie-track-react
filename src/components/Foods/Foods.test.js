import React from 'react';
import { shallow } from 'enzyme';
import Foods from './Foods';

it('should render FoodDashboardPage correctly', () => {
  const wrapper = shallow(<Foods />);
  expect(wrapper).toMatchSnapshot();
})