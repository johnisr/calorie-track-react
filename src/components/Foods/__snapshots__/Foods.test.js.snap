// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render FoodDashboardPage correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Foods />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="row"
>
        <section
                className="section__full-start-end"
        >
                <Connect(Header)
                        history={
                                Object {
                                        "action": "POP",
                                        "block": [Function],
                                        "createHref": [Function],
                                        "go": [Function],
                                        "goBack": [Function],
                                        "goForward": [Function],
                                        "length": 1,
                                        "listen": [Function],
                                        "location": Object {
                                          "hash": "",
                                          "pathname": "/",
                                          "search": "",
                                          "state": undefined,
                                        },
                                        "push": [Function],
                                        "replace": [Function],
                                      }
                        }
                />
        </section>
        <section
                className="section__center-start-end"
        >
                <Connect(FoodListFilters) />
        </section>
        <section
                className="section__center-6-start-end"
        >
                <Connect(FoodListContainer) />
        </section>
        <section
                className="section__center-6-start-end margin-bottom-large"
        >
                <Connect(FoodsDisplayTabs) />
        </section>
</div>,
      "className": "container",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <section
            className="section__full-start-end"
>
            <Connect(Header)
                        history={
                                    Object {
                                                "action": "POP",
                                                "block": [Function],
                                                "createHref": [Function],
                                                "go": [Function],
                                                "goBack": [Function],
                                                "goForward": [Function],
                                                "length": 1,
                                                "listen": [Function],
                                                "location": Object {
                                                  "hash": "",
                                                  "pathname": "/",
                                                  "search": "",
                                                  "state": undefined,
                                                },
                                                "push": [Function],
                                                "replace": [Function],
                                              }
                        }
            />
</section>,
          <section
            className="section__center-start-end"
>
            <Connect(FoodListFilters) />
</section>,
          <section
            className="section__center-6-start-end"
>
            <Connect(FoodListContainer) />
</section>,
          <section
            className="section__center-6-start-end margin-bottom-large"
>
            <Connect(FoodsDisplayTabs) />
</section>,
        ],
        "className": "row",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Connect(Header)
              history={
                            Object {
                                          "action": "POP",
                                          "block": [Function],
                                          "createHref": [Function],
                                          "go": [Function],
                                          "goBack": [Function],
                                          "goForward": [Function],
                                          "length": 1,
                                          "listen": [Function],
                                          "location": Object {
                                            "hash": "",
                                            "pathname": "/",
                                            "search": "",
                                            "state": undefined,
                                          },
                                          "push": [Function],
                                          "replace": [Function],
                                        }
              }
/>,
            "className": "section__full-start-end",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "history": Object {
                "action": "POP",
                "block": [Function],
                "createHref": [Function],
                "go": [Function],
                "goBack": [Function],
                "goForward": [Function],
                "length": 1,
                "listen": [Function],
                "location": Object {
                  "hash": "",
                  "pathname": "/",
                  "search": "",
                  "state": undefined,
                },
                "push": [Function],
                "replace": [Function],
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "section",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Connect(FoodListFilters) />,
            "className": "section__center-start-end",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "section",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Connect(FoodListContainer) />,
            "className": "section__center-6-start-end",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "section",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Connect(FoodsDisplayTabs) />,
            "className": "section__center-6-start-end margin-bottom-large",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "section",
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="row"
>
          <section
                    className="section__full-start-end"
          >
                    <Connect(Header)
                              history={
                                        Object {
                                                  "action": "POP",
                                                  "block": [Function],
                                                  "createHref": [Function],
                                                  "go": [Function],
                                                  "goBack": [Function],
                                                  "goForward": [Function],
                                                  "length": 1,
                                                  "listen": [Function],
                                                  "location": Object {
                                                    "hash": "",
                                                    "pathname": "/",
                                                    "search": "",
                                                    "state": undefined,
                                                  },
                                                  "push": [Function],
                                                  "replace": [Function],
                                                }
                              }
                    />
          </section>
          <section
                    className="section__center-start-end"
          >
                    <Connect(FoodListFilters) />
          </section>
          <section
                    className="section__center-6-start-end"
          >
                    <Connect(FoodListContainer) />
          </section>
          <section
                    className="section__center-6-start-end margin-bottom-large"
          >
                    <Connect(FoodsDisplayTabs) />
          </section>
</div>,
        "className": "container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <section
              className="section__full-start-end"
>
              <Connect(Header)
                            history={
                                          Object {
                                                        "action": "POP",
                                                        "block": [Function],
                                                        "createHref": [Function],
                                                        "go": [Function],
                                                        "goBack": [Function],
                                                        "goForward": [Function],
                                                        "length": 1,
                                                        "listen": [Function],
                                                        "location": Object {
                                                          "hash": "",
                                                          "pathname": "/",
                                                          "search": "",
                                                          "state": undefined,
                                                        },
                                                        "push": [Function],
                                                        "replace": [Function],
                                                      }
                            }
              />
</section>,
            <section
              className="section__center-start-end"
>
              <Connect(FoodListFilters) />
</section>,
            <section
              className="section__center-6-start-end"
>
              <Connect(FoodListContainer) />
</section>,
            <section
              className="section__center-6-start-end margin-bottom-large"
>
              <Connect(FoodsDisplayTabs) />
</section>,
          ],
          "className": "row",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <Connect(Header)
                history={
                                Object {
                                                "action": "POP",
                                                "block": [Function],
                                                "createHref": [Function],
                                                "go": [Function],
                                                "goBack": [Function],
                                                "goForward": [Function],
                                                "length": 1,
                                                "listen": [Function],
                                                "location": Object {
                                                  "hash": "",
                                                  "pathname": "/",
                                                  "search": "",
                                                  "state": undefined,
                                                },
                                                "push": [Function],
                                                "replace": [Function],
                                              }
                }
/>,
              "className": "section__full-start-end",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "history": Object {
                  "action": "POP",
                  "block": [Function],
                  "createHref": [Function],
                  "go": [Function],
                  "goBack": [Function],
                  "goForward": [Function],
                  "length": 1,
                  "listen": [Function],
                  "location": Object {
                    "hash": "",
                    "pathname": "/",
                    "search": "",
                    "state": undefined,
                  },
                  "push": [Function],
                  "replace": [Function],
                },
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": "section",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <Connect(FoodListFilters) />,
              "className": "section__center-start-end",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": "section",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <Connect(FoodListContainer) />,
              "className": "section__center-6-start-end",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": "section",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <Connect(FoodsDisplayTabs) />,
              "className": "section__center-6-start-end margin-bottom-large",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": "section",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
